enterprise/frontend/src/embedding-sdk/types/theme/index.ts

```diff
@@ -126,6 +126,11 @@ export type MetabaseComponentTheme = {
 
   /** Pivot table **/
   pivotTable: {
+    cell: {
```

Comment:
Can you add this to the readme file as well? I only see `cartesian.padding` is added.

Reply:
@WiNloSt Good catch!